portfolio/src/main/webapp/script.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * Fetches a message from the server and adds it to the DOM.
+ */
+async function getMsg() {
+  const response = await fetch('/data');
+  const quote = await response.text();
+  document.getElementById('message-container').innerText = quote;
+}
+
```

Comment:
we usually leave 1 line between functions.

Reply:
Resolved.